apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -175,9 +171,10 @@ export const CreateOrder = () => {
   }
 
   const handleOnSwap = () => {
-    if (createOrder.type === 'limit' && createOrder.limitPrice !== undefined) {
-      const marketPrice = new BigNumber(createOrder.marketPrice)
-      const limitPrice = new BigNumber(createOrder.limitPrice)
+    if (orderData.calculatedPool === undefined) return
```

Comment:
should the button be disabled?

Reply:
I think it is, this is just a function safeguard in case it's called anyway. Plus it quiets typescript in latter lines wondering if it's undefined or not.